File: DFN_Sample_Project_(Android_Java)/app/src/main/java/com/igaworks/dfnsampleproject/DeeplinkActivity.java
package com.igaworks.dfnsampleproject;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import com.igaworks.v2.core.AdBrixRm;

public class DeeplinkActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);

        // This method must call for when app is re-start
        onNewIntent(DeeplinkActivity.this.getIntent());

    }

    @Override
    protected void onNewIntent(Intent intent){
        super.onNewIntent(intent);

        // This method must call for when intent has changed.
        setIntent(intent);

        AdBrixRm.deeplinkEvent(DeeplinkActivity.this);

        // Do View Landing from here

    }
}

File: DFN_Sample_Project_(Android_Kotlin)/app/src/main/java/com/igaworks/dfn_sample_projectkotilin/DeeplinkActivity.kt
package com.igaworks.dfn_sample_projectkotilin

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.igaworks.v2.core.AdBrixRm

class DeeplinkActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // This method must call for when app is re-start
        onNewIntent(this.intent)
    }

    override fun onNewIntent(intent: Intent?) {
        super.onNewIntent(intent)

        // This method must call for when intent has changed.
        setIntent(intent)

        AdBrixRm.deeplinkEvent(this)

        // Do View Landing from here


    }
}